etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrderConverterTest.groovy

```diff
@@ -169,4 +175,91 @@ class HapiOrderConverterTest extends Specification {
         convertedMessageHeader.getEventCoding().getCode() == "O21"
         convertedMessageHeader.getEventCoding().getDisplay().contains("OML")
     }
+
+    def "add contact section to patient resource"() {
```

Comment:
Seems like we should have an additional test for when the mother's maiden name extension doesn't exist since there's an `if` statement that checks for that.

Reply:
Yes, I agree. We worked on the test coverage today and we added a unit test that checks for that.